homotopy-graphics/src/style.rs

```diff
@@ -51,16 +46,16 @@ impl Color {
     // Get color in lightened form. We wrap colors if needed so that they are sufficiently
     // distinguishable for end users.
     //
-    // Colors are calculated from C and R where:
+    // Colors are distinguished as pairs (C, R) where:
     //      C = max(0, D - N - K)
```

Comment:
Could you `println` the values of $C$ for some diagram? $K = max(0, D - N)$ so I have a feeling this is always $0$.

Reply:
In think $K$ is technically not defined a $\max(0, D - N)$. It's actually inferred from the representation of a generator which comes out of the core graphics code.

Reply:
![2022-09-05-082305_786x419_scrot](https://user-images.githubusercontent.com/34209800/188392186-2bf1afd2-9dca-43db-98b8-a4910a2ae2d2.png)

In this case, `Cell 2` gives $C = 0$ but `Cell 0` gives $C = 1$.

`hom` attached: [c_values.zip](https://github.com/homotopy-io/homotopy-rs/files/9487626/c_values.zip)